requirements-dev.txt

```diff
@@ -55,9 +59,14 @@ pytest==7.2.1
 pytz==2022.7.1
 pyyaml==6.0 ; python_version >= '3.6'
 requests==2.28.2 ; python_version >= '3.7' and python_version < '4'
+<<<<<<< HEAD
```

Comment:
As @MVrachev spotted there's a conflict and the tests are failing from the conflict markers left here.
(quick fix running `make requirements`)
After that, I run the tests locally and everything passes!